src/storage/sql_table.cpp

```diff
@@ -108,6 +108,16 @@ void SqlTable::UpdateSchema(const catalog::Schema &schema) {
     }
   }
 
+  // Populate the default value map
```

Comment:
I think we actually need to modify every *other* datatable's default value map.  Specifically, if this schema (1) has a column OID the previous schema version does not and (2) that column comes with a default value, then (3) we need to apply that default value to all previous schema versions to ensure "new" reads will see the retroactive value.

Reply:
We should probably pass in the transaction context and transaction version to this function make this check easier and have a more consistent API across `SqlTable`.

Reply:
> Specifically, if this schema (1) has a column OID the previous schema version does not

I'm using the `DefaultValueMap` of the version as seen by the transaction while filling in default values. So, the previous version's default values don't matter, right? Also, if a new column OID comes in which the previous schema versions don't contain, they shouldn't be made aware of the new column OID.